planning/behavior_path_planner/src/utils/utils.cpp

```diff
@@ -2672,8 +2672,10 @@ PredictedObjects filterObjectsByVelocity(
   PredictedObjects filtered;
   filtered.header = objects.header;
   for (const auto & obj : objects.objects) {
-    const auto v = std::abs(obj.kinematics.initial_twist_with_covariance.twist.linear.x);
-    if (min_v < v && v < max_v) {
+    const auto v_norm = std::hypot(
+      obj.kinematics.initial_twist_with_covariance.twist.linear.x,
+      obj.kinematics.initial_twist_with_covariance.twist.linear.y);
+    if (min_v < v_norm && v_norm < max_v) {
```

Comment:
I agree this change.
But I applied this change in this [PR](https://github.com/autowarefoundation/autoware.universe/pull/4658/files#diff-5e3a3ab9ddbf28c4acb9313e60b7d9f668773f2be8a3e12a1788ef476fce23a6R65-R68)
In this PR, this function is moved to path_safety_checker directory in utils. So could you revert this change?
```suggestion
    const auto v = std::abs(obj.kinematics.initial_twist_with_covariance.twist.linear.x);
    if (min_v < v && v < max_v) {
```